cppcache/src/CacheFactory.cpp

```diff
@@ -123,11 +123,11 @@ Cache CacheFactory::create(
     name = "NativeCache";
   }
 
-  Cache cache = Cache(name, dsProp, ignorePdxUnreadFields,
+  auto cache = Cache(name, dsProp, ignorePdxUnreadFields,
                     pdxReadSerialized, authInitialize);
   cache.m_cacheImpl->setAttributes(attrs);
   try {
-    const char* cacheXml =
+    const auto* cacheXml =
```

Comment:
Should be `const auto` not `const auto*`. I am actually shocked that isn't a warning or an error. Pointerness is auto deduced.

Reply:
Will do.